<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Contingency</class>
 <widget class="QMainWindow" name="Contingency">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contingency Analysis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="mainGridLayout" columnstretch="3,1">
      <item row="2" column="1">
       <widget class="QPushButton" name="applyPushButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="outputGroupBox">
        <property name="title">
         <string>Output</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QGridLayout" name="outputGroupBoxGridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="outputFileNameLabel">
             <property name="text">
              <string>Analysis name</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="outputFileNameLineEdit">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;LSAT will create *name*_ctg.npz in \results\statistics\&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="inputGroupBox">
        <property name="title">
         <string>Input</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QGridLayout" name="inputGroupBoxGridLayout" columnstretch="4,1,1,0">
           <item row="9" column="1">
            <widget class="QToolButton" name="maskRasterToolButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QToolButton" name="attributeTableToolButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QToolButton" name="addRasterToolButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLineEdit" name="maskRasterLineEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="maskRasterLabel">
             <property name="text">
              <string>Mask raster</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" rowspan="6" colspan="3">
            <widget class="QListWidget" name="listWidget"/>
           </item>
           <item row="9" column="2">
            <widget class="QCheckBox" name="referenceCheckBox">
             <property name="text">
              <string>Use project reference as default</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QToolButton" name="removeRasterToolButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Raster datasets</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
